package.json

```diff
@@ -17,14 +17,14 @@
     "pretest": "turbo run pretest",
     "depcheck": "knip --dependencies --tags=-knipignore",
     "knip:fix": "knip --fix",
-    "test": "vitest --run --coverage --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
-    "test:filter": "vitest --run --coverage --config vitest.node.config.mts",
+    "test": "vitest --run --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
+    "test:filter": "vitest --run --config vitest.node.config.mts",
     "test:coverage-merge": "tsx ./scripts/tests-coverage-merge.ts",
     "test:coverage-diff": "tsx ./scripts/tests-coverage-diff.ts",
     "test:watch": "vitest --watch --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
     "test:validate": "./scripts/tests-validate.sh",
-    "test:browser": "vitest --run --coverage --config vitest.browser.config.mts $(scripts/tests-find.sh --browser)",
-    "test:browser:filter": "vitest --run --coverage --config vitest.browser.config.mts",
+    "test:browser": "vitest --run --config vitest.browser.config.mts $(scripts/tests-find.sh --browser)",
+    "test:browser:filter": "vitest --run  --config vitest.browser.config.mts",
```

Comment:
The `--coverage` flag was intended to draw attention to it during development, I think.

Reply:
I'll revert this change before opening for review, thanks for catching this.